controllers/snapshot/snapshot_adapter.go

```diff
@@ -120,29 +120,14 @@ func (a *Adapter) EnsureAllIntegrationTestPipelinesExist() (controller.Operation
 					"integrationTestScenario.Name", integrationTestScenario.Name,
 					"len(integrationPipelineRuns)", len(*integrationPipelineRuns))
 			} else {
-				a.logger.Info("Creating new pipelinerun for integrationTestscenario",
-					"integrationTestScenario.Name", integrationTestScenario.Name)
 				pipelineRun, err := a.createIntegrationPipelineRun(a.application, &integrationTestScenario, a.snapshot)
 				if err != nil {
 					a.logger.Error(err, "Failed to create pipelineRun for snapshot and scenario")
 					return controller.RequeueWithError(err)
 				}
-				a.logger.LogAuditEvent("IntegrationTestscenario pipeline has been created", pipelineRun, h.LogActionAdd,
-					"integrationTestScenario.Name", integrationTestScenario.Name)
 				testStatuses.UpdateTestStatusIfChanged(
 					integrationTestScenario.Name, gitops.IntegrationTestStatusInProgress,
 					fmt.Sprintf("IntegrationTestScenario pipeline '%s' has been created", pipelineRun.Name))
```

Comment:
This is the only piece I'm not sure about.  Why isn't this also being moved to `createIntegrationPipelineRun()`? We would need to update the test status for the scenario in the case of a re-run.  Do you think this is just out of scope?

Reply:
I had the similar question [here](https://redhat-internal.slack.com/archives/C034UFMUSV6/p1695279001912279), the status will be reported [here](https://github.com/redhat-appstudio/integration-service/blob/main/controllers/integrationpipeline/integrationpipeline_adapter.go#L194) as `Inprogress/TestPass/TestFailed` as well, I think it is reasonable to not to add a lot of code here when creating PLR in snapshot and binding controller.